.gitignore

```
@@ -0,0 +1,3 @@
+.idea/
+__pycache__/
+venv/
```

Good add on. FYI, you can ask pycharm/GitHub (or any other IDE) to generate a template for you which is language specific.

You will see that the template is usually much larger and have things like `*.pyc`. Which you added here. These should not be there as it's just a machine version of your files.

Also here, remember to always put a new line at the end of a file. Pycharm should suggest this as well.
```suggestion
venv/

```